content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -15,7 +15,8 @@ continuous integration (CI), and other best practices that improve productivity.
 This means writing a description of which data, ML artifacts, etc. should be in
 the environment at a given time. DVC can restore the <abbr>workspace</abbr>
 files and directories (from a separate data storage) to match that description.
-Here are some benefits of this approach:
+
+## Why do it with DVC
```

Comment:
Is this too salesy? It's nice to have the right-hand CONTENT nav entry.

![image](https://user-images.githubusercontent.com/1477535/97096457-3ba86680-1632-11eb-8352-6f27c3ec5b2b.png)